website/docs/build-apps/how-to-guides/Display-and-filter-chart-data.md

```diff
@@ -0,0 +1,142 @@
+---
+description: This page shows you how to use the Datepicker widget to filter and view Chart data based on specific date ranges.
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+# Plot Chart with Filters
+
+This page shows you how to display and filter data in Chart, using the Select and Datepicker widgets.
+
+
+## Prerequisites
+* A [Chart](/reference/widgets/chart) widget to display data.
+* A query that contains the data you want to filter.
+* [Select](/reference/widgets/select) or [Datepicker](/reference/widgets/datepicker) widget to filter the data.
+
+## Display data in Chart
+
+To display data from a query using the Chart widget, follow these steps:
+
+1. Select a **Chart type** that suits your use case. 
+
+2. Connect the query to the [**Series data**](/reference/widgets/chart#series-data-arrayobject) property of the Chart widget, like:
+
+<dd>
+
+```js
+{{fetchData.data}}
+```
+
+If the query data is not in the expected format, you can use the `map()` function to transform it before passing it to the widget, like:
+
+```js
+{{fetchData.data.map( p => ({x: p.gender, y: p.count}))}}
+```
+
+Use the **Add Series** property to populate the chart with different sets of data.
+
+  <figure>
+  <img src="/img/bar-voter.png" style= {{width:"600px", height:"auto"}} alt=""/>
+  <figcaption align = "center"><i></i></figcaption>
+  </figure>  
+
+
+</dd>
+
+
+
+
+## Filter Chart data 
+
+To filter information based on specific criteria, use widgets like the Datepicker or Select widget for data filtering.
+
+
+<Tabs>
+<TabItem value="select" label="Using Select" default>
+
+ 1. Drag a Select widget, and set the [**Source data**](/reference/widgets/select#source-data-arrayobject) property as required.
```

Comment:
Drop select widget. Also users might not understand what you mean by set the Source data property as required. Clarify